dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -44,7 +44,7 @@ ENV COMPILER="4.05.0"
 # Common OPAM packages
 ENV BASE_OPAM="zarith.1.10 ocamlfind.1.8.1 ounit2.2.2.3 odoc.1.5.2" \
     CI_OPAM="ocamlgraph.1.8.8 yojson.1.7.0" \
-    BASE_ONLY_OPAM="dune.2.7.1 elpi.1.13.5"
+    BASE_ONLY_OPAM="dune.2.7.1 elpi.1.13.6 stdlib-shims.0.1.0"
```

Comment:
I am a bit concerned that this means CI won't catch if we start using Stdlib
OTOH maybe it will since it's not declared in the dune files?

Reply:
In that case I don't think it would be a shame to add stdlib-shims as a dependency of Coq (when compiled with <= 4.08).

But I'm open to to other solutions. I can `opam install stdlib-shims` in the coq-elpi job if you prefer.

Reply:
Running a test to see which hypothesis is right

Reply:
CI catches it, no problem https://gitlab.com/coq/coq/-/jobs/1448166190